README.md

```diff
@@ -104,7 +104,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+The HTML validator cannot validate the ruby code. We could use Nokogiri instead.
 ```
 
```

Comment:
Not exactly, ERB tags (clown hats) are not valid HTML. If you want to grab the code, you can "view source" in the browser and copy the HTML there, which is rendered as a result of Rails processing and replacing the ERB tags.